test/api/manipulation.js

```diff
@@ -496,6 +496,24 @@ describe('$(...)', function() {
     });
   });
 
+  describe('.appendTo', function() {
+
+    it('(Node) : should add element as last child', function() {
+      $('<li class="plum">Plum</li>').appendTo($fruits);
+      expect($fruits.children().eq(3).hasClass('plum')).to.be.ok();
+    });
+
+  });
+
+  describe('.prependTo', function() {
+
+    it('(html) : should add element as first child', function() {
```

Comment:
The test description is incorrect. It describes an HTML-accepting signature, but the test invokes `prependTo` with a Cheerio instance (which is elsewhere described as `"$(...)"`)